tests/test_pipeline_mixin.py

```diff
@@ -140,18 +140,22 @@ def run_task_tests(self, task):
 
             tokenizer_names = []
             processor_names = []
+            commit = None
             if model_arch_name in tiny_model_summary:
                 tokenizer_names = tiny_model_summary[model_arch_name]["tokenizer_classes"]
                 processor_names = tiny_model_summary[model_arch_name]["processor_classes"]
+                commit = tiny_model_summary[model_arch_name]["sha"]
             # Adding `None` (if empty) so we can generate tests
             tokenizer_names = [None] if len(tokenizer_names) == 0 else tokenizer_names
             processor_names = [None] if len(processor_names) == 0 else processor_names
 
             repo_name = f"tiny-random-{model_arch_name}"
 
-            self.run_model_pipeline_tests(task, repo_name, model_architecture, tokenizer_names, processor_names)
+            self.run_model_pipeline_tests(
+                task, repo_name, model_architecture, tokenizer_names, processor_names, commit
+            )
```

Comment:
The change in this PR is just about using the commit hash of tiny models' hub repositories. This was discussed/mentioned briefly by/with @LysandreJik a few month ago.

The goal is to avoid sudden test failure due to some updates on those repos.

Those commit hashes are stored in `tiny_model_summary.json` file.

Reply:
Just a q, why are we not using `revision` instead of the internal `_commit_hash`? Would be cleaner/more understandable to use `revision` which we also use elsewhere imo

Reply:
Very nice point - the only (un-)reason is that I completely forget about `revision`  😭 
Will change it.